packages/fxa-settings/src/lib/channels/helpers.ts

```diff
@@ -2,93 +2,43 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { AccountData } from '../../models';
+import firefox, { FxALoginRequest } from './firefox';
 
-// const ALLOWED_LOGIN_FIELDS = [
-//   'declinedSyncEngines',
-//   'email',
-//   'keyFetchToken',
-//   'offeredSyncEngines',
-//   'sessionToken',
-//   'services',
-//   'uid',
-//   'unwrapBKey',
-//   'verified',
-// ];
-
-// const REQUIRED_LOGIN_FIELDS = [
-//   'email',
-//   'keyFetchToken',
-//   'sessionToken',
-//   'uid',
-//   'unwrapBKey',
-//   'verified',
-// ];
-
-// Might need to go on the Integration?
+// TODO: might need to go on Integration?
 // let uidOfLoginNotification: hexstring = '';
 
-// function hasRequiredLoginFields(loginData) {
-//   const loginFields = Object.keys(loginData);
-//   return (
-//     REQUIRED_LOGIN_FIELDS.filter((field) => !loginFields.includes(field))
-//       .length === 0
-//   );
-// }
-
-/**
- * Get login data from `account` to send to the browser.
- * All returned keys have a defined value.
- */
-// function getLoginData(account: AccountData) {
-// let loginData: Partial<AccountData> = {};
-// for (const key of ALLOWED_LOGIN_FIELDS) {
-//   loginData[key] = account[key];
-// }
-//   // TODO: account for multiservice when we combine reliers
```

Comment:
I'm going to add this comment back since we do need this. I _think_ this has to do with the signup or confirm view since in content-server, `_formatForMultiServiceBrowser` will send `offeredSyncEngines` and `declinedSyncEngines`. It can be looked at in https://mozilla-hub.atlassian.net/browse/FXA-7308.